components/enterprise-contract/download-service.yaml

```diff
@@ -0,0 +1,275 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ec-cli-download-role
+  namespace: enterprise-contract-service
+rules:
+  - apiGroups:
+      - route.openshift.io
+    resourceNames:
+      - ec-cli-download-route
+    resources:
+      - routes
+    verbs:
+      - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ec-cli-download-role-binding
+  namespace: enterprise-contract-service
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ec-cli-download-role
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: enterprise-contract-service
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ec-cli-download-cluster-role
+  namespace: enterprise-contract-service
+rules:
+  - apiGroups:
+      - console.openshift.io
+    resources:
+      - consoleclidownloads
+    verbs:
+      - create
+      - patch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
```

Comment:
Is a ClusterRoleBinding needed because ConsoleCLIDownloads resources are cluster resources?

Reply:
Yes